chia/daemon/keychain_server.py

```diff
@@ -93,6 +97,12 @@ async def add_private_key(self, request: Dict[str, Any]) -> Dict[str, Any]:
                 "error": KEYCHAIN_ERR_KEYERROR,
                 "error_details": {"message": f"The word '{e.args[0]}' is incorrect.'", "word": e.args[0]},
             }
+        except ValueError as e:
+            log.exception(e)
+            return {
+                "success": False,
+                "error": str(e),
+            }
```

Comment:
"Invalid order of mnemonic words" is a ValueError raised by `bytes_from_mnemonic()`. To preserve the error message as-is, the ValueError exception case is handled and simply returns the wrapped error message.